cache.go

```diff
@@ -74,6 +74,16 @@ func (c *Cache[K, V]) Len() int {
 	return c.store.Len()
 }
 
+// Size returns the used size of the cache.
```

Comment:
better to document the behavior of this method so users know it will loop through all entries and may slow down system performance.